src/generated/resources/schema.graphql

```diff
@@ -80,6 +81,17 @@ type DirectSchemaRegistry {
   uri: String!
 }
 
+type FlinkComputePool {
```

Comment:
Minor: should we go with `CCloudFlinkComputePool` for now to be more consistent with the other CCloud types?